app/scripts/script.js

```diff
@@ -0,0 +1,69 @@
+// DateTime Dsplay
+function startTime() {
+    var today = new Date();
+    document.getElementById('datetime').innerHTML =
+     today;
+    var t = setTimeout(startTime, 500);
+  }
+  function checkTime(i) {
+    if (i < 10) {i = "0" + i};  // add zero in front of numbers < 10
+    return i;
+  }
+
+// Function that starts the clock on page load
+  document.getElementById("body").addEventListener("onload", startTime());
```

Comment:
you're actually triggering `startTime` immediately. You aren't waiting. This could be causing issues where if the page takes too long to load, your page crashes.

```suggestion
  document.getElementById("body").addEventListener("onload", () => startTime());
```